db/migrate/20201005134940_fix_directory_application_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class FixDirectoryApplicationColumnName < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :directory_applications, :type, :app_type
```

Comment:
It looks like this name change can just be applied to the original table creation migration